app/build.gradle

```diff
@@ -22,8 +22,8 @@ android {
     def versionMajor = 1
     def versionMinor = 2
     def versionPatch = 0
-    def versionMeta = "-qa.2"
-    def versionCodeIncremental = 1386
+    def versionMeta = "-qa.6"
+    def versionCodeIncremental = 1390
```

Comment:
As now we work together, it is better to bump the version in the release branch after merging `main` into it. For example, I already updated the version to qa.6 (1390) there. Consider this when merging later